pyaedt/edb_grpc/core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,2146 @@
+        self._parent._update_setup()
+
+    @property
+    def basic(self):
+        """Whether if turn on basic adaptive.
+
+        Returns
+        -------
+            ``True`` if bais adaptive is used, ``False`` otherwise.
+        """
+        return self.adaptive_settings.Basic
+
+    @basic.setter
+    def basic(self, value):
+        self.adaptive_settings.Basic = value
+        self._parent._update_setup()
+
+    @property
+    def do_adaptive(self):
+        """Whether if adpative mesh is on.
+
+        Returns
+        -------
+        bool
+            ``True`` if adaptive is used, ``False`` otherwise.
+
+        """
+        return self.adaptive_settings.DoAdaptive
+
+    @property
+    def max_refinement(self):
+        """Maximum number of mesh elements to be added per pass.
+
+        Returns
+        -------
+        int
+        """
+        return self.adaptive_settings.MaxRefinement
+
+    @max_refinement.setter
+    def max_refinement(self, value):
+        self.adaptive_settings.MaxRefinement = value
+        self._parent._update_setup()
+
+    @property
+    def max_refine_per_pass(self):
+        """Maximum number of mesh elementat that can be added during an adaptive pass.
+
+        Returns
+        -------
+        int
+        """
+        return self.adaptive_settings.MaxRefinePerPass
+
+    @max_refine_per_pass.setter
+    def max_refine_per_pass(self, value):
+        self.adaptive_settings.MaxRefinePerPass = value
+        self._parent._update_setup()
+
+    @property
+    def min_passes(self):
+        """Minimum number of passes.
+
+        Returns
+        -------
+        int
+        """
+        return self.adaptive_settings.MinPasses
+
+    @min_passes.setter
+    def min_passes(self, value):
+        self.adaptive_settings.MinPasses = value
+        self._parent._update_setup()
+
+    @property
+    def save_fields(self):
+        """Whether to turn on save fields.
+
+        Returns
+        -------
+        bool
+            ``True`` if save fields is used, ``False`` otherwise.
+        """
+        return self.adaptive_settings.SaveFields
+
+    @save_fields.setter
+    def save_fields(self, value):
+        self.adaptive_settings.SaveFields = value
+        self._parent._update_setup()
+
+    @property
+    def save_rad_field_only(self):
+        """Whether to turn on save radiated fields only.
+
+        Returns
+        -------
+        bool
+            ``True`` if save radiated field only is used, ``False`` otherwise.
+
+        """
+        return self.adaptive_settings.SaveRadFieldsOnly
+
+    @save_rad_field_only.setter
+    def save_rad_field_only(self, value):
+        self.adaptive_settings.SaveRadFieldsOnly = value
+        self._parent._update_setup()
+
+    @property
+    def use_convergence_matrix(self):
+        """Whether to turn on the convergence matrix.
+
+        Returns
+        -------
+        bool
+            ``True`` if convergence matrix is used, ``False`` otherwise.
```

Comment:
```suggestion
        """Whether the convergence matrix is used.

        Returns
        -------
        bool
            ``True`` if the convergence matrix is used, ``False`` otherwise.
```